### ISSUE TITLE
OD radio - Display audio episode image

### ISSUE THREAD

**Issue by RichardPK** [bbc/simorgh#6337]
**Is your feature request related to a problem? Please describe.**
Episode images need to be displayed per-episode. Often - specific episode images are not specified, and so the brand image should be used as a fallback. i.e. See [Afrique image within this design.](https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5e15e72bbc69965446284242)

This image has an impact on the layout of the other elements within the component in its current state.

**Describe the solution you'd like**

The image url is returned in the OD radio endpoint

`"imageUrl": "ichef.bbci.co.uk/images/ic/$recipe/p08b23c8.png"`

The raw ichef image is 16:9

When we display the image on the page we can use either of these ichef recipes to format as square:

http://ichef.bbci.co.uk/images/ic/112x112/p07vj43n.jpg
http://ichef.bbci.co.uk/images/ic/224x224/p07vj43n.jpg

Image will not be displayed on smaller breakpoints, see Zepplin designs.

**A11y acceptance criteria**
Given I am a screenreader user 
When I navigate to the hero image
Then the [alt-text] is announced 

Given I am on a smaller breakpoint
and the image is not visible
When I navigate the page using a screenreader
then the image will not be acknowledged

Content order for screen readers:
The h1 should be announced first
And then the paragraph text is announced,
And that should be followed by the image with the alt-text,
And then the audio player

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
